### DISCUSSION TITLE
GraphQL sprites images

### DISCUSSION THREAD

**Discussion by vJaun** [PokeAPI/pokeapi#713] [Q&A] [answered]
I'm trying to pull images from the GraphQL console but they appear to be null. Where can I pull images of the pokemon from?

```
query samplePokeAPIquery {
  pokemon_v2_pokemonsprites {
    pokemon_id
    pokemon_v2_pokemon {
      id
      name
    }
    sprites
  }
}
```

**Comment by Naramsim** [comment, marked as answer]
Hi, we are aware of the issue. https://github.com/PokeAPI/pokeapi/issues/614

Currently, nobody is working on it. The best would be to use the HTTP API to fetch the sprites.
